dd4hep-toolfile.spec

```diff
@@ -9,35 +9,39 @@ Requires: dd4hep
 %install
 
 mkdir -p %i/etc/scram.d
-cat << \EOF_TOOLFILE >%i/etc/scram.d/dd4hep.xml
-<tool name="dd4hep" version="@TOOL_VERSION@">
+cat << \EOF_TOOLFILE >%i/etc/scram.d/dd4hep-core.xml
+<tool name="dd4hep-core" version="@TOOL_VERSION@">
   <info url="https://github.com/AIDASoft/DD4hep"/>
-  <lib name="DDAlign" />
   <lib name="DDCore" />
-  <lib name="DDCond" />
   <lib name="DDParsers" />
   <client>
-    <environment name="DD4HEP_BASE" default="@TOOL_ROOT@"/>
-    <environment name="LIBDIR" default="$DD4HEP_BASE/lib"/>
-    <environment name="INCLUDE" default="$DD4HEP_BASE/include"/>
+    <environment name="DD4HEP_CORE_BASE" default="@TOOL_ROOT@"/>
+    <environment name="LIBDIR" default="$DD4HEP_CORE_BASE/lib"/>
+    <environment name="INCLUDE" default="$DD4HEP_CORE_BASE/include"/>
   </client>
   <runtime name="ROOT_INCLUDE_PATH" value="$INCLUDE" type="path"/>
-  <runtime name="PATH" value="$DD4HEP_BASE/bin" type="path"/>
+  <runtime name="PATH" value="$DD4HEP_CORE_BASE/bin" type="path"/>
   <use name="root_cxxdefaults"/>
   <use name="root"/>
-  <use name="boost"/> 
+  <use name="boost"/>
   <use name="xerces-c"/>
   <use name="clhep"/>
 </tool>
 EOF_TOOLFILE
 
-cat << \EOF_TOOLFILE >%i/etc/scram.d/dd4hep-cms.xml
-<tool name="dd4hep-cms" version="@TOOL_VERSION@">
-  <lib name="DDCMS" />
-  <client>
-    <environment name="DD4HEP_CMS_BASE" default="@TOOL_ROOT@"/>
-  </client>
-  <use name="dd4hep"/>
+cat << \EOF_TOOLFILE >%i/etc/scram.d/dd4hep.xml
+<tool name="dd4hep" version="@TOOL_VERSION@">
+  <lib name="DDAlign" />
+  <lib name="DDCond" />
+  <use name="dd4hep-core"/>
+</tool>
+EOF_TOOLFILE
+
+cat << \EOF_TOOLFILE >%i/etc/scram.d/dd4hep-geant4.xml
+<tool name="dd4hep-geant4" version="@TOOL_VERSION@">
+  <lib anme="DDG4-static"/>
+  <use name="geant4-core"/>
```

Comment:
@smuzaffar this seems to cause in recent IBs
```
gmake[1]: Entering directory '/build/fabiocos/106X/26301/CMSSW_10_6_X_2019-04-01-2300'
****WARNING: Invalid tool geant4-core. Please fix dd4hep-geant4 file.
```
Should not be geant4core ?

Reply:
thanks @fabiocos for catching this. I have fixed this in https://github.com/cms-sw/cmsdist/pull/4835